.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ jobs:
       - run:
           command: npm install
           name: install dependencies
+      - run:
+          command: npm install -g expo-cli
```

Comment:
Nice work! That's what perseverance and collaboration results in: a working CI pipeline 💪 @banjaminh @Daniel-Gallagher92